File: build_management/bazel_primer/python_tutorial/stage3/main/hello_world.py
from datetime import datetime
import os
import sys
sys.path.append(os.path.realpath('.'))
from main.hello_greet import get_greet
from lib.hello_time import print_localtime

if __name__ == "__main__":
    who = "world"
    print(get_greet(who))
    print_localtime()

File: build_management/bazel_primer/c_tutorial/stage3/main/hello_world.c
#include "lib/hello_time.h"
#include "main/hello_greet.h"
#include <stdio.h>

int main(int argc, char** argv) {
  char* who = "world";
  printf("%s", get_greet(who));
  print_localtime();
  return 0;
}
